demo/src/client/ToDoListPage/ToDoListPage.tsx

```diff
@@ -14,7 +14,7 @@ interface State {
     value?: string
 }
 
-export class App extends React.PureComponent<any, State> {
+export class App extends React.Component<any, State> {
```

Comment:
Was changing this to Component intentional?

Reply:
Yes, the page wasn't re-rendering on a state change to the todo list item isComplete property. Turned out to be because the todoList state object was a List of Items the default PureComponent check wasn't seeing this as a change because it wasn't at the top level. Changing to Component was the simplest fix.

Reply:
Got it.  Sounds good.